.github/workflows/lint.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# From
+# https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/modules/lint_file/lint.yaml.tftpl
+
+name: 'lint'
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: 'lint'
+    runs-on: 'ubuntu-latest'
+    env:
+      EXCLUDE_LINT_DIRS: "\\./anthos-bm-openstack-terraform/resources"
```

Comment:
**Heads-up:**
This `EXCLUDE_LINT_DIRS` value doesn't seem to work.
We still see lint failures in [this lint build](https://github.com/GoogleCloudPlatform/anthos-samples/actions/runs/5126111306/jobs/9220175029?pr=612), in the `anthos-bm-openstack-terraform/resources` folder.